caldav/server.go

```diff
@@ -344,6 +344,8 @@ func (b *backend) Propfind(r *http.Request, propfind *internal.Propfind, depth i
 		return nil, err
 	}
 
+	var compReq CalendarCompRequest
```

Comment:
Style nit: please move this variable declaration right before the `ListCalendarObjects` call, since we don't need it outside that scope.